RELEASE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 ## RELEASE NOTES
 
+### Version 2.60.5-RDM-6350-RC1-prerelease
+**RDM-6635** -  bulk-scanning integration withCCD
```

Comment:
can we change the description to:
`Bulk scanning transaction page integration into CCD web component`

Reply:
done